topics/contributing/tutorials/critical_theory/tutorial.md

```diff
@@ -0,0 +1,223 @@
+>
+> Positivists stress the need for objectivity in the pursuit of knowledge, advocating for the exclusion of personal biases and subjective interpretations. While positivism has significantly influenced the natural sciences, it has also faced criticism for its limitations in addressing subjective experiences, moral values, and aspects of reality that aren't easily quantifiable. Despite these critiques, positivism continues to be a major perspective in the philosophy of science.
+>
+> Max Horkheimer, who played a pivotal role in the **Frankfurt School**, argued that positivism misrepresent human social action by ignoring the influence of socially and historically mediated human consciousness on social facts. Positivism failed to acknowledge that these social facts are not objective entities but are shaped by observers and the broader social context, leading to a distorted view of reality. Secondly, Horkheimer contended that positivism construction of social reality was politically conservative, maintaining the existing status quo rather than challenging it.
+>
+{: .comment}
+
+In his famous work *Dialectics of Ideology and Technology*, the American sociologist Alvin Gouldner pointed out that, in contrast to this superficial and clearly insufficient conception, **critical thinking must be capable of questioning even the logic of reasoning itself**, that is, it must be capable of problematising what has hitherto been treated as self-evident, of turning into an object of reflection what had previously been simply a tool ({% cite Gouldner1976 %}).
+
+> <comment-title>Frankfurt School and critical thinking </comment-title>
+>
+> Some critical theorists of the first generation, including Max Horkheimer, Theodor Adorno and Walter Benjamin, argued that in Western democracies the capacity for critical reasoning is rapidly being eclipsed.
+>
+> Pointing to developments such as the intrusion of the state, the culture industry and the concentration of wealth in fewer and fewer people, these thinkers fear that the ideological and material conditions that make public interaction and critical thinking possible are being undermined by the increasing standardisation, fragmentation and commercialisation of everyday life.
+>
+{: .comment}
+
+From the assumption of this conception of critical thinking a new question arises: is it really in the interest of education system to encourage critical thinking? One element that can help to answer this question is the hidden curriculum.
+
+> <question-title>Critical thinking quiz</question-title>
+>
+> 1. **How does critical thinking align with the principles of both Critical Theory and Critical Learning Theory?** <br>
+> &nbsp; a. By avoiding critical examination of societal issues. <br>
+> &nbsp; b. By promoting conformity to existing structures. <br>
+> &nbsp; c. By fostering a reflective approach to learning and encouraging critical analysis. <br>
+> &nbsp; d. By discouraging diverse perspectives.
```

Comment:
```suggestion
> 1. **How does critical thinking align with the principles of both Critical Theory and Critical Learning Theory?**
>    a. By avoiding critical examination of societal issues.
>    b. By promoting conformity to existing structures.
>    c. By fostering a reflective approach to learning and encouraging critical analysis.
>    d. By discouraging diverse perspectives.
```

please use proper markdown formatting here. We can style it as a/b/c/d with CSS which feels a bit safer.

For this page we could add a bit like

```
<style>
blockquote.question ol ol li {
  list-style: lower-alpha;
}
</style>
```

in order to achieve that.

Also it's a *lot* of bold, maybe conider not bolding the questions?